.readthedocs.yaml

```diff
@@ -11,3 +11,5 @@ sphinx:
 python:
   install:
     - requirements: docs/requirements.txt
+
+formats: [pdf]
```

Comment:
Do you need html here also or will readthedocs build the html target?

Reply:
Readthedocs will build the html target by default, this is just for additional formats you wish to build. You can see the other formats [here](https://docs.readthedocs.io/en/stable/config-file/v2.html#formats).